Test/AwsCommonRuntimeKitTests/XCBaseTestCase.swift

```diff
@@ -35,20 +38,44 @@ extension XCTestCase {
     }
 
     func skipIfiOS() throws {
```

Comment:
Debatable: These functions are unused, apart from `skipIfLinux`. We can just get rid of them until needed.

Reply:
We will use them soon for IoT related tests. I will keep them here for now.